cmd/mesh/manifest-versions.go

```diff
@@ -105,8 +103,25 @@ func getVersionCompatibleMap(versionsURI string, binVersion *goversion.Version,
 		}
 	}
 	if myVersionMap == nil {
-		l.logAndFatal("This operator version ", binVersion.String(),
-			" was not found in the global manifestVersions map.")
+		return nil, fmt.Errorf("this operator version %s was not found in the version map", binVersion.String())
+	}
+	return myVersionMap, nil
+}
+
+func loadCompatibleMapFile(versionsURI string, l *logger) (b []byte, err error) {
+	if util.IsHTTPURL(versionsURI) {
+		b, err = httprequest.Get(versionsURI)
```

Comment:
nit: it's best to avoid named return values where possible for improved readability through the smallest possible scoping, and also to remain consistent with project style. also, it's normal to redefine err as a local var where possible. 
here, i propose changing to:
if b, err :=  httprequest.Get(versionsURI); err == nil {
  return b, nil
}

Reply:
Sure, fixed.